src/styles/smartscalar-workaround.css

```diff
@@ -2,30 +2,53 @@
  * WORKAROUND. To be removed after fine-grained theming is available.
  **/
 .smartscalar [data-testid="scalar-value"] {
-  font-weight: 200;
   text-align: left;
-
-  /** Force alignment. */
-  margin-left: -2px;
 }
 
 .smartscalar [data-testid="scalar-period"] {
   display: none !important;
 }
 
-.smartscalar [data-card-key]>div:has(div[data-testid="scalar-container"]) {
-  align-items: flex-start;
-}
-
 .smartscalar [data-testid="scalar-previous-value"] > div > h4 {
-  color: #ADABA9;
   font-weight: 400;
 }
 
-.smartscalar [data-testid="scalar-previous-value"] > div > h4 > span {
-  color: #ADABA9;
+body[data-theme="luminara"] {
+  .smartscalar [data-testid="scalar-value"] {
+    font-weight: 200;
+    font-family: "Libre Caslon Display";
+  }
+}
+
+body[data-theme="stitch"] {
+  .smartscalar [data-card-key]>div:has(div[data-testid="scalar-container"]) {
```

Comment:
Not pretty, but we have to use this workaround until the SDK officially supports fine-grained theming.

Reply:
@deniskaber I agree with this being misleading. Let me check with Alberto what he thinks. I think Alessio's goals for the 3 themes was to [show what would be eventually possible](https://metaboat.slack.com/archives/C063Q3F1HPF/p1719336477507209?thread_ts=1719295515.987739&cid=C063Q3F1HPF) with the SDK, so this might be a necessary evil.

> these, on purpose, are pushing the possibilities of what we could do (e.g., gradients, multiple fonts)